src/api/cli/create_rawtx.go

```diff
@@ -531,14 +531,31 @@ func makeChangeOut(outs []wallet.UxBalance, chgAddr string, toAddrs []SendAmount
 		return nil, err
 	}
 
-	if haveChange {
-		outAddrs = append(outAddrs, mustMakeUtxoOutput(chgAddr, changeAmount, changeHours))
-	}
-
+	var spendCoinsAmt uint64
 	for i, to := range toAddrs {
+		// check if changeHours > 0
+		// if changeHours is zero then all addrHours will also be zero
```

Comment:
if changeHours is zero, then all remaining coin hours will be evenly split into dest addresses.